src/Components/Posts.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Posts = () => {
+    return (
+        <div>
+            Posts
```

Comment:
Here you could have invoked your post.js component. so:

`<div>
     <Post/>
     <Post/>
     <Post/>
</div>`

You would also need to import the post.js file